craft_cli/helptexts.py

```diff
@@ -357,9 +347,14 @@ def _build_markdown_command_help(self, usage, overview, options, other_command_n
         overview = process_overview_for_markdown(overview)
         textblocks.append(f"## Summary:\n\n{overview}")
 
-        option_lines = ["## Options:"]
+        option_lines = [
+            "## Options:",
+            "| | |",
+            "|-|-|",
+        ]
         for title, text in options:
-            option_lines.extend(_build_item_markdown(title, text))
+            option_lines.append(f"| `{title}` | {text} |")
```

Comment:
maybe a future change could be to make this a 3 column line so the long options and short ones are aligned and would be easy to spot which one has which, I understand this is a bigger change in the help parser though

Reply:
Yes! I opened #114 to track it.